net/uradvd/files/uradvd.config

```diff
@@ -6,6 +6,10 @@ config interface
 	option ifname 'br-lan'
 	# lifetime of the default route (in seconds)
 	option default_lifetime '0'
+	option valid_lifetime '86400'
+	option preferred_lifetime '14400'
+	option max_router_adv_interval '600'
+	option min_router_adv_interval '200'
```

Comment:
Are these the default options already?

Reply:
Yes.
https://github.com/freifunk-gluon/uradvd/blob/main/uradvd.c#L44-L52